src/Components/FormCreation/CreateEditFormHeader.tsx

```diff
@@ -86,6 +91,9 @@ function CreateEditFormHeader({
           updateHeader();
         }}
       />
+      <Button color="primary" startIcon={<FormSettingsOutlineIcon />}>
+        Form Settngs
```

Comment:
typo! Also, the button doesn't do anything onClick. Should it open a popup / new page?